### ISSUE TITLE
Boon of Hysh + Bolstering boon + Wild Healing

### ISSUE THREAD

**Issue by th3gatekeeper** [WarEmu/WarBugs#6479]
For reference:
http://waronlinebuilder.org/#career=am;11;0;0;7:11:;;;3456:;0:0:0:0:

Those two tactics + Boon of Hysh spamming before combat actually starts in scenarios (say while capping the first BO) can boost players morales up above morale 2. Add some morale/sec gear and you can pop M3s within just a few seconds of actually hitting combat - basically ensuring you will win the initial encounter.

It seems this is an abuse of the abilities and Bolstering boon should only work if it actually healed the player? Or another possible smaller fix would be if it rather than gave a flat 250 morale, but rather Morale/Sec - like 250 over 5 seconds. Preventing the ability to spam the heal on someone to get morale up super super fast. (Id also be in favor of seeing a similar thing on Chosen's tactic "destined for victory as well)

Ive seen it abused in a few scenarios allowing Knights to pop their morale 3 "No Escape" which hits everyone for 1200 damage AND roots for 10 seconds AND cannot be broken. A pretty massive imbalance when paired with AMs spamming Boon of Hysh with these tactics pre-combat, giving a HUGE advantage on the initial interaction - which in many BO games makes a big difference..

**Comment by Evildoor** [comment]
I would like to see video demonstration of this. I logged onto AM on dev server, slotted these two tactics and spammed BoH on myself - i haven't managed to even get to M1 out of combat. Ok, maybe it's possible with 2+ AMs - however, i don't see how this can work at first BO, since you'll lose all morale while riding to the second one (if we are talking Nordenwatch/GoE). Maybe it is possible if you stay on BO and wait for enemy to attack you.
Side note - DoK morale pump was removed from the game for exactly this reason, but DoK had morale pump on 0.5 second cast, here we are talking about 2 second cast, and AM's morale tactic was considered garbage because of this. But this is just memories, not proof of anything.
